18F/identity-idp
# LG-6348: Implement "Ready to verify" page content

**Why**: So that a user has the instructions they need to proof their identity in person.

**Testing Instructions:**

1. Go to http://localhost:3000
2. Sign in
3. Go to http://localhost:3000/verify
4. Complete proofing flow up to document capture
5. Submit with [failing document](https://developers.login.gov/testing/#document-upload) as image upload
6. Use troubleshooting option to verify in person
7. Complete proofing flow
8. Observe page as shown in screenshot below 

**Screenshot:**

![localhost_3000_verify_in_person_ready_to_verify (2)](https://user-images.githubusercontent.com/1779930/178557200-5729dd81-24e4-4702-baa4-0690c7521a3b.png)
